.github/workflows/build_and_test.sh

```diff
@@ -15,5 +15,6 @@ if [[ -z $(git status -s) ]]; then
     # TODO: Revise this check once we can reliably build without changes
     # exit 1
 fi
+
+# Note this doesn't run the fuzz tests in scripts/build_fuzz.sh
```

Comment:
This comment seems unnecessary... It makes sense as part of the PR but doesn't make much sense in the code imo.